Movieee/Movieee/UIs/Home Screen/Cell/HomeScreenMovieListCollectionViewCell.swift

```diff
@@ -0,0 +1,11 @@
+import UIKit
+
+class HomeScreenMovieListCollectionViewCell: UICollectionViewCell {
+    @IBOutlet weak var homeScreenMoviePoster: UIImageView!
+    @IBOutlet weak var homeScreenMovieName: UILabel!
```

Comment:
```suggestion
final class HomeScreenMovieListCollectionViewCell: UICollectionViewCell {
    @IBOutlet private weak var posterImageView: UIImageView!
    @IBOutlet private weak var nameLabel: UILabel!
```
tên Cell có vẻ hơi dài nhỉ, thu gọn hơn được không?

Reply:
Cái này em không để private được anh nhé, không là em ko nhận được bên TableviewCell